### PR TITLE
BUGZ-1046 - send entity packets reliably for initial query

### PR DESCRIPTION
**Pull request by roxanneskelly**
https://highfidelity.atlassian.net/browse/BUGZ-1046

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20704/HighFidelity-Beta-PR16017-a991b6d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20704/HighFidelity-Quest-Beta-PR16017-a991b6d.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20704/HighFidelity-Beta-PR16017-a991b6d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20704/HighFidelity-Beta-Interface-PR16017-a991b6d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20704/HighFidelity-Beta-PR16017-a991b6d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20704/HighFidelity-Beta-Interface-PR16017-a991b6d.exe

File: assignment-client/src/entities/EntityTreeSendThread.cpp [hunk 1]
```diff
@@ -160,6 +160,11 @@ bool EntityTreeSendThread::traverseTreeAndSendContents(SharedNodePointer node, O
     if (sendComplete && nodeData->wantReportInitialCompletion() && _traversal.finished()) {
         // Dealt with all nearby entities.
         nodeData->setReportInitialCompletion(false);
+        // initial stats and entity packets are reliable until the initial query is complete
+        // to guarantee all entity data is available for safe landing/physics start.  Afterwards
+        // the packets are unreliable for performance.
+        nodeData->stats.getStatsMessage().setReliable(false);
+        nodeData->getPacket().setReliable(false);
```
Comment (roxanneskelly) [inline assignment-client/src/entities/EntityTreeSendThread.cpp]: the packets are initially created as reliable in NodeData::init

**Comment by roxanneskelly** [comment]
Marking as DNM as we're going to have more people look at this.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20786/HighFidelity-Beta-PR16017-2e5127f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20786/HighFidelity-Beta-Interface-PR16017-2e5127f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20786/HighFidelity-Beta-PR16017-2e5127f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20786/HighFidelity-Beta-Interface-PR16017-2e5127f.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20786/HighFidelity-Beta-PR16017-2e5127f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20786/HighFidelity-Quest-Beta-PR16017-2e5127f.apk)

**Comment by SimonWalton-HiFi** [comment]
Seems unfortunate that we have to make a copy of each packet to send it, even after we've changed to unreliable service. Probably not a major cost though.

**Pull request merged**